adapters/base/src/main/java/org/apache/cassandra/sidecar/adapters/base/CassandraStorageOperations.java

```diff
@@ -213,4 +214,11 @@ public void outOfRangeDataCleanup(@NotNull String keyspace, @NotNull String tabl
         jmxClient.proxy(StorageJmxOperations.class, STORAGE_SERVICE_OBJ_NAME)
                  .forceKeyspaceCleanup(concurrency, keyspace, table);
     }
+
+    @Override
+    public GossipStatusResponse gossipStatus()
+    {
+        return new GossipStatusResponse(jmxClient.proxy(StorageJmxOperations.class, STORAGE_SERVICE_OBJ_NAME)
+                                                 .isGossipRunning() ? "RUNNING" : "NOT_RUNNING");
```

Comment:
What do you think about moving `"RUNNING"` and `"NOT_RUNNING"` to an enum?

Reply:
I actually agree with @yifan-c  suggestion. So, what about having the enum for the `"OK"` and `"NOT_OK"`?